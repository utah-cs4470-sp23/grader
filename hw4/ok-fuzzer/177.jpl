print "AgK"
time type U = float
type C = bool
assert [ v ( .3 , 33 , ( 1254655 ) ) , true ], ""
let V = if true then K ( { { [ false ] } } ) else .3
write image R ( sum [ g : [ 23 , array [  ] U ] ] d ) to "<z"
let A [ c ] = { true [ false ] }
write image sum [  ] w (  ) to "g"
let {  } = if false then K (  ) else c ( array [  ] true )
show array [ G : if true then ( { F ,  } ) else p { 6 }, C : sum [  ] true ] true
read image "s*" to o [ j ]
let {  } = 147
let Q = sum [ t : .421170 ] q
write image true to ""
let {  } = k
print "I"
type e = float [ , , , ,  ] [ , ,  ] [  ]
show [ n { 17 } ]
assert [ false ,  ] { 62 }, ""
write image array [ S : { .59 { 2 } }, t : 1 { 6 },  ] {  } to ""
read image "" to x
assert false, "e"
print ""
let Y [ D ] = 9
assert g, "]1C"
type f = float
write image if sum [  ] [  ] then 46 else D to ""
show array [ q : sum [ d : ( 7 ) ] false ] true [ R ( false ) ]
print ""
show 06 [ sum [  ] { [ [ .528 ] ,  ] } ,  ]
assert sum [  ] {  } { 588 }, ""
type I = int
read image "%" to O [ g ]
time read image "" to e [ M ]
let {  } = ( { sum [ N : u ( 7 ) ] false } )
type z = p
assert false, ")r"
assert sum [ C : 17,  ] 0, ""
assert { false } { 2 }, ""
write image [ array [  ] ( array [ r : r ( 8 ) ] E ) , .47 ,  ] to ""
print "p"
read image "" to i
let { g } = c
type t = { int }
show array [ n : false ] [ A , array [  ] true , { 177634 } ] { 98 }
type J = float
print ""
assert a (  ), ""
read image "" to I [ h , z , G , v ]
print "p"
read image "ZOtz" to Z [ D , k , o ]
time type g = float
let M = 07 [ L ]
assert sum [  ] 2, ""
type W = int [ , , ,  ]
let m = { 4 , true }
time read image "" to u
read image "~ZQP&" to C [ p , W , V , b ]
print ""
let { c } = h
type i = bool
type y = k
show sum [ d : 29 { 9 } ] true
write image [ 7 ] to ""
assert u ( [ 0 ,  ] ), ""
let {  } = true
show if false then array [ b : ( [ l ] ) ] { .0 } else false
write image array [  ] true to ""
assert true, "PO"
time assert V ( true , { [ false ] } ,  ), "aV"
type o = int
time write image { {  } } to ""
assert ( if sum [  ] true then true else false ), ""
show T (  )
time type r = bool
show sum [  ] ( true )
time read image "" to r [ C , y , n ]
print "4ZBk"
write image l to "LLS9"
