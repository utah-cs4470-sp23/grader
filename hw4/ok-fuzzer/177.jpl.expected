(PrintCmd "AgK")
(TimeCmd (TypeCmd U (FloatType)))
(TypeCmd C (BoolType))
(AssertCmd
 (ArrayLiteralExpr
  (CallExpr v (FloatExpr 0) (IntExpr 33) (IntExpr 1254655))
  (TrueExpr))
 "")
(LetCmd
 (ArgLValue (VarArgument V))
 (IfExpr
  (TrueExpr)
  (CallExpr
   K
   (TupleLiteralExpr (TupleLiteralExpr (ArrayLiteralExpr (FalseExpr)))))
  (FloatExpr 0)))
(WriteCmd
 (CallExpr
  R
  (SumLoopExpr
   g
   (ArrayLiteralExpr (IntExpr 23) (ArrayLoopExpr (VarExpr U)))
   (VarExpr d)))
 "<z")
(LetCmd
 (ArgLValue (ArrayArgument A c))
 (TupleLiteralExpr (ArrayIndexExpr (TrueExpr) (FalseExpr))))
(WriteCmd (SumLoopExpr (CallExpr w)) "g")
(LetCmd
 (TupleLValue)
 (IfExpr (FalseExpr) (CallExpr K) (CallExpr c (ArrayLoopExpr (TrueExpr)))))
(ShowCmd
 (ArrayLoopExpr
  G
  (IfExpr
   (TrueExpr)
   (TupleLiteralExpr (VarExpr F))
   (TupleIndexExpr (VarExpr p) 6))
  C
  (SumLoopExpr (TrueExpr))
  (TrueExpr)))
(ReadCmd "s*" (ArrayArgument o j))
(LetCmd (TupleLValue) (IntExpr 147))
(LetCmd (ArgLValue (VarArgument Q)) (SumLoopExpr t (FloatExpr 0) (VarExpr q)))
(WriteCmd (TrueExpr) "")
(LetCmd (TupleLValue) (VarExpr k))
(PrintCmd "I")
(TypeCmd e (ArrayType (ArrayType (ArrayType (FloatType) 5) 3) 1))
(ShowCmd (ArrayLiteralExpr (TupleIndexExpr (VarExpr n) 17)))
(AssertCmd (TupleIndexExpr (ArrayLiteralExpr (FalseExpr)) 62) "")
(WriteCmd
 (ArrayLoopExpr
  S
  (TupleLiteralExpr (TupleIndexExpr (FloatExpr 0) 2))
  t
  (TupleIndexExpr (IntExpr 1) 6)
  (TupleLiteralExpr))
 "")
(ReadCmd "" (VarArgument x))
(AssertCmd (FalseExpr) "e")
(PrintCmd "")
(LetCmd (ArgLValue (ArrayArgument Y D)) (IntExpr 9))
(AssertCmd (VarExpr g) "]1C")
(TypeCmd f (FloatType))
(WriteCmd
 (IfExpr (SumLoopExpr (ArrayLiteralExpr)) (IntExpr 46) (VarExpr D))
 "")
(ShowCmd
 (ArrayLoopExpr
  q
  (SumLoopExpr d (IntExpr 7) (FalseExpr))
  (ArrayIndexExpr (TrueExpr) (CallExpr R (FalseExpr)))))
(PrintCmd "")
(ShowCmd
 (ArrayIndexExpr
  (IntExpr 6)
  (SumLoopExpr
   (TupleLiteralExpr (ArrayLiteralExpr (ArrayLiteralExpr (FloatExpr 0)))))))
(AssertCmd (SumLoopExpr (TupleIndexExpr (TupleLiteralExpr) 588)) "")
(TypeCmd I (IntType))
(ReadCmd "%" (ArrayArgument O g))
(TimeCmd (ReadCmd "" (ArrayArgument e M)))
(LetCmd
 (TupleLValue)
 (TupleLiteralExpr (SumLoopExpr N (CallExpr u (IntExpr 7)) (FalseExpr))))
(TypeCmd z (VarType p))
(AssertCmd (FalseExpr) ")r")
(AssertCmd (SumLoopExpr C (IntExpr 17) (IntExpr 0)) "")
(AssertCmd (TupleIndexExpr (TupleLiteralExpr (FalseExpr)) 2) "")
(WriteCmd
 (ArrayLiteralExpr
  (ArrayLoopExpr (ArrayLoopExpr r (CallExpr r (IntExpr 8)) (VarExpr E)))
  (FloatExpr 0))
 "")
(PrintCmd "p")
(ReadCmd "" (VarArgument i))
(LetCmd (TupleLValue (ArgLValue (VarArgument g))) (VarExpr c))
(TypeCmd t (TupleType (IntType)))
(ShowCmd
 (ArrayLoopExpr
  n
  (FalseExpr)
  (TupleIndexExpr
   (ArrayLiteralExpr
    (VarExpr A)
    (ArrayLoopExpr (TrueExpr))
    (TupleLiteralExpr (IntExpr 177634)))
   98)))
(TypeCmd J (FloatType))
(PrintCmd "")
(AssertCmd (CallExpr a) "")
(ReadCmd "" (ArrayArgument I h z G v))
(PrintCmd "p")
(ReadCmd "ZOtz" (ArrayArgument Z D k o))
(TimeCmd (TypeCmd g (FloatType)))
(LetCmd (ArgLValue (VarArgument M)) (ArrayIndexExpr (IntExpr 7) (VarExpr L)))
(AssertCmd (SumLoopExpr (IntExpr 2)) "")
(TypeCmd W (ArrayType (IntType) 4))
(LetCmd (ArgLValue (VarArgument m)) (TupleLiteralExpr (IntExpr 4) (TrueExpr)))
(TimeCmd (ReadCmd "" (VarArgument u)))
(ReadCmd "~ZQP&" (ArrayArgument C p W V b))
(PrintCmd "")
(LetCmd (TupleLValue (ArgLValue (VarArgument c))) (VarExpr h))
(TypeCmd i (BoolType))
(TypeCmd y (VarType k))
(ShowCmd (SumLoopExpr d (TupleIndexExpr (IntExpr 29) 9) (TrueExpr)))
(WriteCmd (ArrayLiteralExpr (IntExpr 7)) "")
(AssertCmd (CallExpr u (ArrayLiteralExpr (IntExpr 0))) "")
(LetCmd (TupleLValue) (TrueExpr))
(ShowCmd
 (IfExpr
  (FalseExpr)
  (ArrayLoopExpr
   b
   (ArrayLiteralExpr (VarExpr l))
   (TupleLiteralExpr (FloatExpr 0)))
  (FalseExpr)))
(WriteCmd (ArrayLoopExpr (TrueExpr)) "")
(AssertCmd (TrueExpr) "PO")
(TimeCmd
 (AssertCmd
  (CallExpr V (TrueExpr) (TupleLiteralExpr (ArrayLiteralExpr (FalseExpr))))
  "aV"))
(TypeCmd o (IntType))
(TimeCmd (WriteCmd (TupleLiteralExpr (TupleLiteralExpr)) ""))
(AssertCmd (IfExpr (SumLoopExpr (TrueExpr)) (TrueExpr) (FalseExpr)) "")
(ShowCmd (CallExpr T))
(TimeCmd (TypeCmd r (BoolType)))
(ShowCmd (SumLoopExpr (TrueExpr)))
(TimeCmd (ReadCmd "" (ArrayArgument r C y n)))
(PrintCmd "4ZBk")
(WriteCmd (VarExpr l) "LLS9")
